man/systemd-vmspawn.xml

```diff
@@ -132,6 +132,18 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--qemu-kernel=</option><replaceable>PATH</replaceable></term>
```

Comment:
same here, i#d drop the "qemu" prefix. 

in fact we should do that for most other options, unless they directly expose a qemu-specific concept that makes no sense if we ever supported some other backend one day. (not saying that we ever actually want to support any other backend, but i think we should keep the options open at least in the interface)

Reply:
Fixed in 37e75950ff.